spec/system/targets/show_spec.rb

```diff
@@ -383,6 +398,85 @@
     end
   end
 
+  scenario "student submits form on a target anonymously" do
+    sign_in_user student.user, referrer: target_path(target_l3)
+
+    # This target should have a 'Submit Form' section.
+    find(".course-overlay__body-tab-item", text: "Submit Form").click
```

Comment:
I think the tab title needs to change for discussion enabled assignments. Currently, I don't think students are adequately notified that their submission will be shared with their peers.

- This tab should probably read `Submit for discussion`.
- It should maybe also say `Submit for discsussion & review` - if space allows, in that particular case.
- The _Submit_ button within the form should probably read _Submit and share_ when discussion is enabled.

@pupilfirst/designers Can you think of other tweaks we can make to indicate the _public_ nature of work submitted via this form?

Reply:
@vinutv LGTM!